Makefile

```diff
@@ -98,6 +98,10 @@ release-test:
 	mkdir -p $(builddir)/release
 	cd $(builddir)/release && cmake -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=release $(topdir) && $(MAKE) && $(MAKE) test
 
+release-test-ci:
+	mkdir -p $(builddir)/release
+	cd $(builddir)/release && cmake -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=release -D BUILD_SHARED_LIBS=ON $(topdir) && $(MAKE) && MONERO_PARALLEL_TEST_JOBS=nproc $(topdir)/tests/run-tests-parallel.sh
```

Comment:
Notice, that I smuggled -D BUILD_SHARED_LIBS=ON, in order to leverage faster linkage for the release version.

Reply:
The target has a default of nproc, and the runner script also has a default of nproc, if the env var isn't set. Then it tries to find the minimum between nproc (or the forced setting) and the **currently** reasonable value of 2 jobs.